setup.py

```diff
@@ -8,19 +8,50 @@
 
 import datetime
 import os
-
 from os.path import (
     dirname,
     join as opj,
+    sep as pathsep,
+    splitext,
 )
-from setuptools import Command, DistutilsOptionError
-from setuptools.config import read_configuration
-
+import setuptools
```

Comment:
isn't this a duplicate import? (see lines 20 and following)

Reply:
I see one usage, in line 306; `if Version(setuptools.__version__) >= Version('38.6.0'):`

(that being said, setuptools 38.6.0 is March 2018, so positively ancient)